marklogic/src/test/scala/quasar/physical/marklogic/xml/ToDataSpec.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2014–2016 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.physical.marklogic.xml
+
+import quasar.Predef._
+import quasar.Data._
+
+class ToDataSpec extends quasar.Qspec {
```

Comment:
Should probably add one that uses namespaced element/attribute names to confirm or deny my suspicions about those.